packages/p2p/src/components/my-ads/my-ads.jsx

```diff
@@ -1,33 +1,47 @@
 import React, { Component, Fragment } from 'react';
 import { Button }                     from '@deriv/components';
+import Dp2pContext                    from 'Components/context/dp2p-context';
 import { localize }                   from 'Components/i18next';
+import { requestWS }                  from 'Utils/websocket';
 import FormAds                        from './form-ads.jsx';
 import { MyAdsTable }                 from './my-ads-table.jsx';
-// import ToggleAds                      from './toggle-ads.jsx';
+import ToggleAds                      from './toggle-ads.jsx';
 import './my-ads.scss';
 
 class MyAds extends Component {
     state = {
-        ad_id    : '',
-        // is_enabled: false,
-        show_form: false,
+        ad_id     : '',
+        is_enabled: false,
+        show_form : false,
+        is_loading: true,
     };
 
     handleShowForm = (show_form) => {
         this.setState({ show_form });
     };
 
+    componentDidMount() {
+        requestWS({ p2p_agent_info: 1 }).then(response => {
+            if (!response.error) {
+                this.setState({ is_enabled: !!response.p2p_agent_info.is_active, is_loading: false });
```

Comment:
`response.p2p_agent_info.is_active` is always going to be 0 or 1, so we don't need the `!!` the concern would be if they send it as a string instead of a number, so if you are worried you can `+` instead

Reply:
is_enabled that we send to `ToggleAds` component is expecting `boolean` type

Reply:
then we should do `!!+response.p2p_agent_info.is_active`